### DISCUSSION TITLE
Github Support hasn't responded to urgent ticket.

### DISCUSSION THREAD

**Discussion by calledblake** [community/community#141084] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My school github account was suspended for no apparent reason and I have all of my school projects that need to be turned in on those repos. I made a ticket asking for to reinstate the account or even just tell me why it was suspended, yet there is no response. I've paid for GitHub pro for a couple years now and I've never met a more unprofessional company when it comes to support.

**Comment by calledblake** [comment]
I didn't even get an email saying I got suspended, it just shows up when I log in.

**Comment by ebndev** [comment]
Hi @BlakeMoran14,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
